test/unit_tests/test_eeg_classifier.py

```diff
@@ -119,3 +120,23 @@ def test_predict_trials():
     with pytest.warns(UserWarning, match="This method was designed to predict "
                                          "trials in cropped mode."):
         clf.predict_trials(MockDataset(), return_targets=False)
+
+
+def test_eeg_classifier_clonable():
+    preds = np.array(
+        [
+            [[0.2, 0.1, 0.1, 0.1], [0.8, 0.9, 0.9, 0.9]],
+            [[1.0, 1.0, 1.0, 1.0], [0.0, 0.0, 0.0, 0.0]],
+            [[1.0, 1.0, 1.0, 0.2], [0.0, 0.0, 0.0, 0.8]],
+            [[0.9, 0.8, 0.9, 1.0], [0.1, 0.2, 0.1, 0.0]],
+        ]
+    )
+    clf = EEGClassifier(
```

Comment:
Shouldn't we put here some callbacks as parameter to check whether we can clone something with callbacks? Here callbacks is None, so nothing will happen in the `_uniquely_named_callbacks`?